mist/runtime/loss.py

```diff
@@ -31,6 +31,64 @@ def forward(self, y_true, y_pred):
         loss = torch.mean(loss)
         return loss
 
+#Implement clDice:
+class soft_cldice(nn.Module):
+    def __init__(self, iter_=3, smooth = 1., exclude_background=False):
```

Comment:
Be consistent about your spacing with equal signs. As a general rule of thumb, I don't use spaces in arguments and use them when defining or setting values. For example,

def __init__(self, iter_=3, smooth=1):
  self.iter = iter_

Also, why do you use a variable name iter_? Maybe make this clearer with more descriptive variable names like iterations?